cep-releases.md

```diff
@@ -0,0 +1,72 @@
+<table>
+<tr><td> Title </td><td> Conda Release Schedule </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Ken Odegard &lt;kodegard@anaconda.com&gt; </td></tr>
+<tr><td> Created </td><td> May 20, 2022 </td></tr>
+<tr><td> Updated </td><td> May 20, 2022 </td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Describes a release cadence for all future `conda` versions starting with 5.0.0.
+
+## Specification
+
+We propose a monthly release schedule where the tagging/release occurs during the week (per ISO 8601, Monday through Sunday) of the 15th.
```

Comment:
```suggestion
We propose a bi-monthly release schedule where the tagging/release occurs during the week (per ISO 8601, Monday through Sunday) of the 15th.
```

You mention a monthly release schedule in the alternatives below:

> 2. Follow a monthly schedule.
     - Rejected for being too fast for multi-user installs.